name,value,relevant log,explanation
io.bytes.per.checksum,-315110497,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-315110497, effective=null
	",value-hit
dfs.datanode.fsdatasetcache.max.threads.per.volume,4,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-315110497, effective=null
	",value-hit
yarn.timeline-service.flowname.max-size,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-315110497, effective=null
	",value-hit
io.bytes.per.checksum,-315110497,"ERROR	Task: attempt_1701312551309_0684_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-315110497, effective=null	",value-hit
dfs.datanode.fsdatasetcache.max.threads.per.volume,4,"ERROR	Task: attempt_1701312551309_0684_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-315110497, effective=null	",value-hit
yarn.timeline-service.flowname.max-size,0,"ERROR	Task: attempt_1701312551309_0684_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-315110497, effective=null	",value-hit
io.bytes.per.checksum,-315110497,"INFO	Diagnostics report from attempt_1701312551309_0684_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-315110497, effective=null	",value-hit
dfs.datanode.fsdatasetcache.max.threads.per.volume,4,"INFO	Diagnostics report from attempt_1701312551309_0684_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-315110497, effective=null	",value-hit
yarn.timeline-service.flowname.max-size,0,"INFO	Diagnostics report from attempt_1701312551309_0684_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-315110497, effective=null	",value-hit
io.bytes.per.checksum,-315110497,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,-315110497,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,-315110497,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,-315110497,INFO	Exiting with status 1: java.lang.NullPointerException,property-item-hit	io
